blaze/compute/tests/test_sql_compute.py

```diff
@@ -1691,6 +1692,118 @@ def test_isin():
     assert normalize(result_sql_expr) == normalize(expected)
 
 
+def test_not_isin_literal():
+    result = t[~t.name.isin(['foo', 'bar'])]
```

Comment:
While creating the failing test for an `isin` of a single column select, I noticed that this `result` should be `expr`. Will change.